desc/compute/geom_utils.py

```diff
@@ -94,19 +94,19 @@ def rpz2xyz(pts):
 
 
 def xyz2rpz_vec(vec, x=None, y=None, phi=None):
-    """Transform vectors from cartesian (X,Y,Z) to polar (R,phi,Z) form.
+    """Transform from cartesian (X,Y,Z) to polar (R,phi,Z) form.
```

Comment:
This was still correct, no need to change. same for rpz2xyz_vec

Reply:
This is more precise.